### PR TITLE
deps: cookie@0.4.0

### PR DESCRIPTION
**Pull request by smaclell**
Bump cookie to 0.4.0 in order to support SameSite=None

### CODE REVIEW

**Comment by dougwilson** [comment]
Thanks! Does this change this module's API? If so, can the README be updated to reflect any API changes?

**Comment by smaclell** [comment]
Updated! Sorry for missing that.

File: README.md [hunk 1]
```diff
@@ -101,10 +101,10 @@ Specifies the `boolean` or `string` to be the value for the `SameSite` `Set-Cook
   - `true` will set the `SameSite` attribute to `Strict` for strict same site enforcement.
   - `false` will not set the `SameSite` attribute.
   - `'lax'` will set the `SameSite` attribute to `Lax` for lax same site enforcement.
+  - `'none'` will set the `SameSite` attribute to `None` for an explicit cross-site cookie.
   - `'strict'` will set the `SameSite` attribute to `Strict` for strict same site enforcement.
 
-More information about the different enforcement levels can be found in the specification
-https://tools.ietf.org/html/draft-west-first-party-cookies-07#section-4.1.1
+More information about the different enforcement levels can be found in [the specification](https://tools.ietf.org/html/draft-ietf-httpbis-rfc6265bis-03#section-4.1.2.)
```
Comment (dougwilson) [inline README.md]: The anchor doesn't seem to be working o  this link when rendered in github

Comment (smaclell) [reply README.md]: I will update this. Thank you for your patience.

Comment (smaclell) [reply README.md]: I have updated the link and tested it using the Github Preview. It now takes you to the correct section.

**Comment by smaclell** [comment]
Thanks again for all your help on this PR. I believe all the docs should now be up to date. We have tested this with one of our services in a staging environment and everything worked swimmingly.

I am going to be completely offline for until around July 28th, but let me know if there is anything else you would like so that I can address it when I return. Have a great day and thanks for all the awesome open source work you do!

**Comment by NilsEngelbach** [comment]
Please Merge this Pull Request to enable SameSite=None Cookies.

At the moment an error is thrown when using `SameSite: "None"` Cookie Option.

`TypeError: option sameSite is invalid`

**Comment by NilsEngelbach** [comment]
@dougwilson Thanks for merging this! Will there also be a new version released?

**Comment by Moniarchy** [comment]
NPM still has the older version of the cookie module 0.3.1: https://unpkg.com/express-session@1.16.2/package.json

Are you planning to publish this change to NPM? With the Chrome breaking change to Same-site defaults, this change might be urgently needed.

**Pull request merged**